packages/cli/src/__tests__/utils.test.ts

```diff
@@ -112,18 +112,10 @@ describe('printConfigLintTotals', () => {
     expect(redColoretteMocks).toHaveBeenCalledWith('❌ Your config has 1 error.\n');
   });
 
-  it('should print warnign and error', () => {
-    printConfigLintTotals({ ...totalProblemsMock, warnings: 2 });
-    expect(process.stderr.write).toHaveBeenCalledWith(
-      '❌ Your config has 1 error and 2 warnings.\n'
-    );
-    expect(redColoretteMocks).toHaveBeenCalledWith('❌ Your config has 1 error and 2 warnings.\n');
-  });
-
```

Comment:
We have only one rule or configuration, so there can only be one severity, `error` or `warning`